notification/src/handler/notification/notification.handler.js

```diff
@@ -64,8 +64,8 @@ async function updatePaymentWithRepeater (payment, notification, ctpClient) {
           + ` max retries. Error: ${JSON.stringify(serializeError(err))}`)
       }
       /* eslint-disable-next-line no-await-in-loop */
-      currentPayment = await ctpClient.fetchById(ctpClient.builder.payments, currentPayment.id)
-      currentPayment = currentPayment.body.results[0] // eslint-disable-line prefer-destructuring
+      const response = await ctpClient.fetchById(ctpClient.builder.payments, currentPayment.id)
+      currentPayment = response.body // eslint-disable-line prefer-destructuring
       currentVersion = currentPayment.version
```

Comment:
is it guaranteed that we will never have an undefined error for `currentPayment.version`?

Reply:
Nothing is guaranteed 100% in life, and it's the same for this case. It could be that the payment does not exist anymore for some reason and we would get null for `currentPayment`. However this would mean that the payment was deleted in between (this code is in the place where you have 409 error) and I cannot think of a reason why it could happen. It's just theory. So for now, I'd say it's ok to have it like this. WDYT?